mappings/net/minecraft/util/random/RandomGenerator.mapping

```diff
@@ -0,0 +1,26 @@
+CLASS net/minecraft/unmapped/C_rlomrsco net/minecraft/util/random/RandomGenerator
+	METHOD m_afzjxjzq nextInt (I)I
+		ARG 1 bound
+	METHOD m_bggirwfo createPositionalRandomFactory ()Lnet/minecraft/unmapped/C_coqzjjrt;
+	METHOD m_culopwlx skip (I)V
+		ARG 1 count
+	METHOD m_djbqzhqs nextFloat ()F
+	METHOD m_ezprnxnd nextInt ()I
+	METHOD m_fakwfybt nextDouble ()D
+	METHOD m_ifdmmgky nextLong ()J
+	METHOD m_imwpekuz createUnsafe ()Lnet/minecraft/unmapped/C_rlomrsco;
+	METHOD m_iwfdczgh setSeed (J)V
+		ARG 1 seed
+	METHOD m_lzlwtgyn nextGaussian ()D
+	METHOD m_mkdxraps nextBoolean ()Z
+	METHOD m_ngqtmirh createLegacy (J)Lnet/minecraft/unmapped/C_rlomrsco;
+		ARG 0 seed
+	METHOD m_paekwdby derive ()Lnet/minecraft/unmapped/C_rlomrsco;
```

Comment:
What does this do?

Reply:
this mapping already existed, git didnt catch the file rename

Reply:
Ah